packages/react-hooks/README.md

```diff
@@ -208,26 +220,6 @@ const MyComponent = () => {
 });
 ```
 
-## useMountSync (deprecated)
```

Comment:
I removed the docs for these deprecated hooks because if we don't want people using them, there's no reason to call them out in the package docs.